tests/python/test_define.py

```diff
@@ -5,36 +5,50 @@
 import torch
 
 from nvfuser import FusionDefinition
-
-
-def test_define_tensor_contiguous():
-    with FusionDefinition() as fd:
-        inp = fd.define_tensor([2, 3], contiguity=True)
-        out = fd.ops.add(inp, inp)
-        fd.add_output(out)
-
-    inp_tensor = torch.randn(2, 3, device="cuda")
-    out_tensor = fd.execute([inp_tensor])[0]
-    torch.testing.assert_close(out_tensor, inp_tensor * 2)
-
-
-def test_define_tensor_noncontiguous():
-    with FusionDefinition() as fd:
-        inp = fd.define_tensor([2, 3])
-        out = fd.ops.add(inp, inp)
-        fd.add_output(out)
-
-    inp_tensor = torch.randn(8, device="cuda").as_strided([2, 3], [4, 1])
-    out_tensor = fd.execute([inp_tensor])[0]
-    torch.testing.assert_close(out_tensor, inp_tensor * 2)
-
-
-def test_define_tensor_broadcast():
-    with FusionDefinition() as fd:
-        inp = fd.define_tensor([1, 2, 1], contiguity=True)
-        out = fd.ops.add(inp, inp)
-        fd.add_output(out)
-
-    inp_tensor = torch.randn(1, 2, 1, device="cuda").as_strided([1, 2, 1], [0, 1, 0])
-    out_tensor = fd.execute([inp_tensor])[0]
-    torch.testing.assert_close(out_tensor, inp_tensor * 2)
+from utils import NVFuserTest
+
+
+# I tried to merge the tests to opinfo and
+# [failed](https://github.com/NVIDIA/Fuser/commit/b0ccb481a7d60f944abf4b6b164f625fec31d147).
+# There seems to be a bug in opinfo causing the generated test to pass fewer
+# arguments than parameters.
```

Comment:
hmmm... that sounds strange. Thanks for logging it here. I wonder if we should have this as an issue as well?

Reply:
Done!